functions/Remove-SqlOrphanUser.ps1

```diff
@@ -181,49 +186,150 @@ Will remove from all databases the user OrphanUser EVEN if exists their matching
+
+                                        if ($NumberObjects -gt 0)
+                                        {
+                                            if ($Force)
+                                            {
+                                                Write-Verbose "Parameter -Force was used! The schema '$($sch.Name)' have $NumberObjects underlying objects. We will change schema owner to 'dbo' and drop the user."
+
+                                                if ($Pscmdlet.ShouldProcess($db.Name, "Changing schema '$($sch.Name)' owner to 'dbo'. -Force used."))
+				                                {
+                                                    $AlterSchemaOwner += "ALTER AUTHORIZATION ON SCHEMA::[$($sch.Name)] TO [dbo]`r`n"
+
+                                                    $results += [pscustomobject]@{
+                                                                                    Instance = $server.Name
+                                                                                    Database = $db.Name
+                                                                                    SchemaName = $sch.Name
+                                                                                    Action = "ALTER OWNER"
+                                                                                    SchemaOwnerBefore = $sch.Owner
+                                                                                    SchemaOwnerAfter = "dbo"
+                                                                                }
+                                                }
+                                            }
+                                            else
+                                            {
+                                                Write-Warning "Schema '$($sch.Name)' owned by user $($User.Name) have $NumberObjects underlying objects. If you want to change the schemas' owner to 'dbo' and drop the user anyway, use -Force parameter. Skipping user '$USer'"
+                                                $SkipUser = $true
+                                                break
+                                            }
+                                        }
+                                        else
+                                        {
+                                            if ($sch.Name -eq $User.Name)
+                                            {
+                                                Write-Verbose "The schema '$($sch.Name)' have the same name as user '$($User.Name)'. Schema will be dropped."
+
+                                                if ($Pscmdlet.ShouldProcess($db.Name, "Dropping schema '$($sch.Name)'."))
+                                                {
+                                                    $DropSchema += "DROP SCHEMA [$($sch.Name)]"
+
+                                                    $results += [pscustomobject]@{
```

Comment:
If no further manipulation of the object is needed, there is no reason to collect it in $results,
just put it in the output right away